server/datastore/mysql/software.go

```diff
@@ -2592,7 +2592,10 @@ last_vpp_install AS (
 						SELECT
 							COUNT(*) AS count_installer_labels,
 							COUNT(lm.label_id) AS count_host_labels,
-							SUM(CASE WHEN lbl.created_at IS NOT NULL AND :host_label_updated_at >= lbl.created_at THEN 1 ELSE 0 END) as count_host_updated_after_labels
+							SUM(CASE 
+							WHEN lbl.created_at IS NOT NULL AND lbl.label_membership_type = 0 AND :host_label_updated_at >= lbl.created_at THEN 1
+							WHEN lbl.created_at IS NOT NULL AND lbl.label_membership_type = 1 THEN 1
```

Comment:
The additional case handles a situation where a label is
- manual (not based on a query)
- exclude_any

since iOS and iPadOS hosts never get their `label_updated_at` field updated, the previous case would always fail.